node/tessel-export.js

```diff
@@ -90,59 +101,90 @@ Tessel.Port = function(name, socketPath, board) {
   this.sock.on('readable', function() {
     var queued;
 
+    // Copy incoming data into the reply buffer
     replyBuf = Buffer.concat([replyBuf, this.sock.read()]);
```

Comment:
Just spotted this line, which indicates that the patch needs to be rebased, take a look at master: https://github.com/tessel/t2-firmware/blob/master/node/tessel-export.js#L101-L103